docs/expo/localization.md

```
@@ -1,400 +1,400 @@
+_Learn about getting started and configuring localization in an Expo project using expo-localization._
+
+If you want your app to be easy to use for users who speak different languages or come from different cultures, you should localize it. Localizing an app makes it adapt to the locale of the user's device. The app will show translations and currencies that the user knows and understands. Numbers, lists, and more will be formatted in a way that the user is used to.
+
+This guide uses the `expo-localization` library for accessing user language settings and adding support for multiple languages. It uses `i18n-js` as an example to add multi-language support.
+
+## Getting the user's language
+
+Use the [`expo-localization`](https://docs.expo.dev/versions/latest/sdk/localization/) library to get the user's current language. Install the package by running the following command:
+
+```bash
+$ npx expo install expo-localization
```

markdownlint-cli2 (0.18.1)

Dollar signs used before commands without showing output

(MD014, commands-show-output)